WalletWasabi.Gui/Global.cs

```diff
@@ -133,6 +133,20 @@ public Global(string dataDir, string torLogsFile, Config config, UiConfig uiConf
 		public async Task InitializeNoWalletAsync()
 		{
 			InitializationStarted = true;
+
+			#region ProcessKillSubscription
```

Comment:
These regions lead only to very long code. Why not use a simple method - something like `RegisterDisposeHandlers`.

Reply:
ACK